dotcom-rendering/src/web/components/Island.tsx

```diff
@@ -20,13 +20,21 @@ interface ClientOnlyProps {
 	expediteLoading?: boolean;
 }
 
+interface InteractionProps {
+	deferUntil: 'interaction';
+	clientOnly?: false;
+	placeholderHeight?: never;
+	children: JSX.Element;
+	expediteLoading?: false;
+}
```

Comment:
No - We always need a server-rendered version because (as it works now) the client must expose some kind of UI for the user to interact with, which would trigger hydration!